MojioSDK/mojio-sdk-model/src/main/java/io/moj/mobile/android/sdk/model/utils/TimeFormatHelpers.java

```diff
@@ -0,0 +1,97 @@
+package io.moj.mobile.android.sdk.model.utils;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.PeriodFormatter;
+import org.joda.time.format.PeriodFormatterBuilder;
+
+import java.util.Arrays;
+
+/**
+ * Util class used to format DateTime to and from the server time format.
+ * Created by mhorie on 16-01-15.
+ */
+public class TimeFormatHelpers {
+
+    private static final String FORMAT_FROM_SERVER = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+    private static final String FORMAT_TO_SERVER = "yyyy-MM-dd\'T\'HH:mm:ss.SSS\'Z\'";
+    private static final DateTimeFormatter FORMATTER_FROM_SERVER = DateTimeFormat.forPattern(FORMAT_FROM_SERVER);
```

Comment:
JodaTime's DateTimeFormatters are immutable so one thing I think would be really handy is a getter for these formatters directly - could be useful when dealing with time zones on the device